app.vue

```diff
@@ -1,3 +1,18 @@
 <template>
-  <NuxtPage />
-</template>
\ No newline at end of file
+  <UContainer>
+    <UHorizontalNavigation :links="links" class="border-b border-gray-200 dark:border-gray-800 mb-4" />
+    <NuxtPage />
+  </UContainer>
+</template>
+<script lang="ts" setup>
+const links = [{
+  label: 'Posts',
+  icon: 'i-heroicons-home',
+  to: '/'
+},
+{
+  label: 'Employee',
+  icon: 'i-heroicons-user',
+  to: '/employee'
+}]
```

Comment:
please always prefer standard route naming,
like 
url | description
-- | -- |
/posts | index page (listing of post)
/posts/create | create new post
 /posts/{post} | show post details
 /posts/{post}/edit | edit post

Reply:
resolved